webrtc-stats.html

```diff
@@ -464,6 +465,23 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>remoteTimestamp</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMHighResTimeStamp</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Present if <code>isRemote</code> is <code>true</code>,
+                  <code>remoteTimestamp</code>, of type
+                  <code>DOMHighResTimeStamp</code> [[!HIGHRES-TIME]],
+                  represents the remote timestamp at which these statistics
```

Comment:
I think we should be explicit that this value only present on inbound streams when isRemote is true.